aviatrix/resource_aviatrix_remote_syslog.go

```diff
@@ -0,0 +1,179 @@
+package aviatrix
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func resourceAviatrixRemoteSyslog() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixRemoteSyslogCreate,
+		Read:   resourceAviatrixRemoteSyslogRead,
+		Update: resourceAviatrixRemoteSyslogUpdate,
+		Delete: resourceAviatrixRemoteSyslogDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"index": {
```

Comment:
Shouldn't `index` be ForceNew? For example, if I change the `index` from 0 to 3. I would expect my config for index 0 would be removed, instead you end up with both 0 and 3 configured.